### ISSUE TITLE
Trouble with the script

### ISSUE THREAD

**Issue by rstreeter** [numixproject/numix-folders#113]
I have tried using the script and I get this error
> python -u "gui"
Traceback (most recent call last):
  File "gui", line 359, in <module>
    win = NumixFoldersGUI(style, colour, colour_list)
  File "gui", line 159, in __init__
    self.colourentries[i].set_max_width_chars(7)
AttributeError: 'Entry' object has no attribute 'set_max_width_chars'
> exit status: 256

I also have installed it from the PPA and get an error 

> bash "numix-folders"
numix-folders: line 104: ./gui/gui: No such file or directory
> exit status: 0

I am curious how to get this working?

**Comment by wa4557** [comment]
What desktop environment? You have to run a very old GTK version if you're getting this error message...

**Comment by rstreeter** [comment]
I am running Xfce 4.10

**Comment by bil-elmoussaoui** [comment]
@rstreeter create a new python file and paste this, run the python file and share the result here 
```python
from gi.repository import Gtk
print(Gtk.get_major_version() , Gtk.get_minor_version())
```

**Comment by rstreeter** [comment]
When I run it I get this 
> python -u "new.py"
(
3L, 12L)
> exit status: 0

**Comment by bil-elmoussaoui** [comment]
Try using `python3 new.py`

**Comment by rstreeter** [comment]
Here is the output I got from that
robert@robert-NerdHeadRepair:~$ python3 new.py
3 12
robert@robert-NerdHeadRepair:~$

**Comment by bil-elmoussaoui** [comment]
Well, if you the same way to execute the gui file i think it should work ;)

**Comment by rstreeter** [comment]
No luck let me check my other computer running the same exact system.

**Comment by rstreeter** [comment]
Still no luck. I know I have used this script before and had no problem.

**Comment by wa4557** [comment]
You can run the script using numix-folders -t until this bug is fixed

Robert <notifications@github.com> schrieb am Mi., 26.08.2015, 0:37:

**Comment by rstreeter** [comment]
Thank you @wa4557 that worked like a charm.

**Comment by neitsab** [comment]
**tl;dr:** numix-folders `.deb` files on Launchpad don't contain `/opt/numix-folders/gui`, while their `.tar.gz` counterparts do. Direct link to examples: [14.04](https://launchpad.net/~numix/+archive/ubuntu/ppa/+files/numix-folders_1.0.2%2B201508281632%7E174%7Eubuntu14.04.1_all.deb) [15.10](https://launchpad.net/~numix/+archive/ubuntu/ppa/+files/numix-folders_1.0.2%2B201508281632%7E174%7Eubuntu15.10.1_all.deb).

---
Today I stumbled upon the same `/gui/gui: No such file or directory` error on a friend's freshly dist-upgraded Ubuntu GNOME 14.04. Steps to reproduce:

```
$ sudo add-apt-repository ppa:numix/ppa -y && sudo apt update
$ sudo apt install numix-folders
$ numix-folders
```

I didn't use bash or python explicitly, just the regular `numix-folders` command that got installed in `$PATH`.

I used `locate` to see where the package was installed (`/opt/numix-folders/`), went there and realized the `gui` subdirectory was indeed missing. So I checked the package archive ([direct link](https://launchpad.net/~numix/+archive/ubuntu/ppa/+files/numix-folders_1.0.2%2B201508281632%7E174%7Eubuntu14.04.1.tar.gz)), which did contain this folder... Only to realize I was checking the `.tar.gz` and not the `.deb` file.

Thanks for your amazing work!

**Comment by PaoloRotolo** [comment]
Hi guys!
I've just uploaded a new version that should fix the issue. It's actually building on Launchpad.

Let me know.

**Comment by Foggalong** [comment]
Thanks @PaoloRotolo \O/

**Comment by wa4557** [comment]
@PaoloRotolo Building seemed to fail with that error message: `cp: cannot stat 'debian/tmp/numix-folders.desktop': No such file or directory`

**Comment by bil-elmoussaoui** [comment]
This is fixed in my case, but still waiting for the latest modifications

**Comment by PaoloRotolo** [comment]
@wa4557 run
``sudo apt-get update``` first. Be sure to download latest version from Launchpad.

**Comment by wa4557** [comment]
@PaoloRotolo Ok building did work but there's still this error: 

`/bin/bash: \ncd /opt/numix-folders && ./numix-folders "$1": File or directory not found` (the line break doesn't work obviously)

**Comment by Foggalong** [comment]
@PaoloRotolo @wa4557 This is actually working for me now! You still having problems?

**Comment by wa4557** [comment]
Yes works fine now :)

Joshua H. Fogg <notifications@github.com> schrieb am Do., 24.09.2015, 13:41:

**Comment by Foggalong** [comment]
Thanks @PaoloRotolo :+1:

**Comment by PaoloRotolo** [comment]
:+1:
